requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
```

Comment:
Thanks for reminding me of wikipedia. Flask is being done seperately as I specify a flask env.